test/classifier/test.cxx

```diff
@@ -533,7 +533,7 @@ struct ClassifierTest
             totalWeight = std::accumulate(a, a + data.ClassIter(ii).size(),
                                           totalWeight);
             std::transform(tmp.begin(), tmp.end(),tmp.begin(), 
-                           std::bind(std::divides<double>(), totalWeight, std::placeholders::_2));
+                           std::bind(std::divides<double>(), std::placeholders::_1, totalWeight));
```

Comment:
This is very likely the only one where the position mattered. Not sure why the other stuff didn't segfault on most platforms.

Maybe the tests were never ran?

Reply:
The old implementation with bind2nd() was battle-tested and correct, but maybe that's not what you mean.

Do you mean that I may have prematurely merged #500 without checking?  That could have been the case, sorry about that.  Usually, I reviewed the code and at least ran the test suite, but I remember that at least once, that was not possible due to technical problems on my end.  In this case, since the change was already shipped via conda-forge, I may have assumed that it was safe enough.

Reply:
Right. That is what I mean. Also, i think that `_2` didn't exist and that only `_1` exists. The test suite revealed that.

Reply:
Yeah.... We skip the tests at conda forge, and likely only the c library is used.